api/controllers/artists.py

```diff
@@ -3,8 +3,15 @@
 from sqlalchemy.orm import Session
 from api.database import get_db
 from api.schemas import ArtistSchema
-from api.models import Artist, User, PaymentProvider, PaymentUrl
-from api.schemas import ArtistBase, PaymentCreate, ArtistFull
+from api.models import (
+    Artist,
+    User,
+    PaymentProvider,
+    PaymentUrl,
+    SocialMedia,
+    SocialLink,
+)
+from api.schemas import ArtistBase, PaymentCreate, ArtistFull, SocialCreate
```

Comment:
(2/4) This goes with my earlier comment on this file:
```suggestion
from api.schemas import ArtistSchema, ArtistBase, PaymentCreate, ArtistFull, SocialCreate
```